MockDataNaming.md

```diff
@@ -94,6 +106,10 @@ with the fallback base name
 ```
 POST|-login-|169d720631e603967135cfce10d235e94aac22b87500ea09d1be295f5b300dca
 ```
+and an optional wildcard base name
+```
+POST|-login-|*
```

Comment:
So if wildcards are enabled, this will cause anything that would hit the `POST|-login-` endpoint, with any data, to get the same file? What happens if you want to for example say "I want the username to be x but the password can be a wildcard?" Thinking of this in relation to something where dates can be whatever but where other things you want specified.